config/overlays/kind-tests/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
where is this overlay getting applied?

Reply:
checkout the [make file](https://github.com/opendatahub-io/data-science-pipelines-operator/pull/384/files/ce4d075a3f7c57458f85928d6f92ac9fcfff50fe#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R158)